Dockerfile

```diff
@@ -2,7 +2,16 @@ FROM google/cloud-sdk:latest
 
 RUN apt-get update
 RUN apt-get --yes install openjdk-11-jre
+RUN apt-get --yes install picard-tools
```

Comment:
Can combine all these installs into one line `RUN apt-get --yes install openjdk-11-jre picard-tools` and we should specify the versions tool=version.

Reply:
I noticed that the downside of using one "RUN apt-get" line, is that when you add one tool, docker needs to rebuild all.

Reply:
Docker has this concept of "layers" that represent atomic units of change to a base image. If I remember properly each line in the Dockerfile maps to a layer, so this makes sense.